src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
```

Comment:
Don't use names, such as result for your variables
```suggestion
  const stateHistory = [];
```